views/github-create-branch.hbs

```diff
@@ -21,15 +21,47 @@
 <input type="hidden" id="_csrf" name="_csrf" value="{{csrfToken}}" />
 <input type="hidden" id="jiraHost" name="jiraHost" value="{{jiraHost}}" />
 
-{{> navigation hideBackButton=true }}
-
 <section class="gitHubCreateBranch">
   <div class="headerImage">
     <img src="/public/assets/jira-and-github.png" alt="Jira and GitHub logos" />
   </div>
   <div class="gitHubCreateBranch__title">
-    Create a Branch
+    <div class="gitHubCreateBranch__header">Create a Branch</div>
```

Comment:
Just checked the designs, this should be Create GitHub branch

<img width="304" alt="Screen Shot 2022-09-01 at 4 20 09 pm" src="https://user-images.githubusercontent.com/37155488/187845434-0c00d56c-0331-464b-b7df-14629539b0e0.png">

Reply:
Updated